packages/webapp/components/layouts/ProfileLayout/index.tsx

```diff
@@ -148,7 +156,9 @@ export default function ProfileLayout({
     }
   }, [profile]);
 
-  if (isFallback && !initialProfile) {
+  const isReadyProfile = isCurrentUserProfile && !isReady;
```

Comment:
I think the variable name is misleading. If the profile is ready, I am expecting to see the content not an empty page. You can use the suggestion below or something similar.
```suggestion
  const isLoadingReferral = isCurrentUserProfile && !isReady;
```

Reply:
Agree, I think your suggestion is quite good so going with that